admin-client/pom.xml

```diff
@@ -131,6 +132,7 @@
                                                 org/keycloak/util/EnumWithStableIndex.java,
                                                 org/keycloak/util/JsonSerialization.java,
                                                 org/keycloak/util/SystemPropertiesJsonParserFactory.java,
+                                                org/keycloak/util/Time.java,
```

Comment:
I don't understand this one. It looks there is no class `org.keycloak.util.Time` in keycloak-core module of Keycloak codebase? And the class `org.keycloak.common.util.Time` from keycloak-common is already included above.

Reply:
Copy paste typo